docs/technical-overview.md

```diff
@@ -3,10 +3,33 @@ title: Technical Overview
 sidebar_label: Technical Overview
 ---
 
-_This page is a stub._
+## Architecture
+
+This project follows `react-native-web` philosophy of ["Write Once, Render Anywhere"](https://www.youtube.com/watch?v=HLWM2uhv2wI). The web version is using
+[Next.js](https://github.com/zeit/next.js/) and the mobile version is powered by
+[Expo](https://github.com/expo/expo).
 
 ## Technologies
 
-## Architecture
+- [React Native](https://facebook.github.io/react-native/) - JS framework for writing real, natively rendering mobile applications for iOS and Android.
```

Comment:
`Natively rendered` maybe (instead of rendering), outside of that LGTM

Reply:
Thanks, sounds definitely better. Updated 🚀